prov/usnic/src/usdf_cq.c

```diff
@@ -462,6 +463,9 @@ usdf_cq_post_soft(struct usdf_cq_hard *hcq, void *context, size_t len,
 	}
 
 	cq->c.soft.cq_last_op = USDF_SOFT_CQ_WRITE;
+
+	/* TODO: How do I report this if it fails?!?!? */
+	write(cq->object.fd, &val, sizeof(val));
```

Comment:
Are you just trying to wake up the listener?  If so, why not a uint8_t (vs. a uint64_t)?